source/adapters/level_zero/CMakeLists.txt

```diff
@@ -73,36 +72,36 @@ target_include_directories(LevelZeroLoader-Headers
 
 add_ur_adapter(${TARGET_NAME}
     SHARED
-    ${L0_DIR}/ur_interface_loader.cpp
```

Comment:
@veselypeta , @kbenzie : in intel/llvm, we fetch the L0 loader with a specific tag for the headers:

sycl/plugins/level_zero/CMakeLists.txt
```cpp
  set(LEVEL_ZERO_LOADER_REPO "https://github.com/oneapi-src/level-zero.git")
  set(LEVEL_ZERO_LOADER_TAG v1.11.0)
```

I dont see it here. If we added it here then now we need to be careful that both unified-runtime and intel/llvm keep the same L0 loader tag always. Or maybe https://github.com/intel/llvm/pull/11252 can have a change where the L0 loader tag used by the unified-runtime is grabbed in a CMake file in intel/llvm

@smaslov-intel : please see above.

Reply:
Fetching L0 loader already exists in the same cmake a few lines above: https://github.com/oneapi-src/unified-runtime/blob/adapters/source/adapters/level_zero/CMakeLists.txt#L11

Reply:
ah ok, thanks @pbalcer